projects/inc/saconfig.h

```diff
@@ -51,13 +51,13 @@ constexpr DACConfig dac1cfg1 = {
 };
 constexpr const DACConfig* KOLLM_DAC_CFG = &dac1cfg1;
 
-/* moment of inertia of steering assembly about the steer axis, kg-m^2 */
-constexpr float STEER_ASSEMBLY_INERTIA_WITH_WEIGHT = 0.1942f; /* with 1 kg weight plate on each side of cross bar*/
-constexpr float STEER_ASSEMBLY_INERTIA_WITHOUT_WEIGHT = 0.0828f; /* without weight plates */
-constexpr float STEER_ASSEMBLY_INERTIA = STEER_ASSEMBLY_INERTIA_WITH_WEIGHT; /* default configuration */
+/* Moment of inertia of steering assembly about the steer axis, kg-m^2 */
+constexpr float STEER_ASSEMBLY_INERTIA_WITH_WEIGHT = 0.1942f; // with 1 kg weight plate on each side of cross bar
```

Comment:
https://github.com/oliverlee/phobos/blob/2dba991be83c8153f664a57f89b4b6faa0cacdda/projects/inc/simbicycle.h#L49

although this gets changed a bit in a pending PR